.github/workflows/shared_test.yml

```diff
@@ -17,6 +17,9 @@ on:
         required: true
       NEXUS_PASSWORD:
         required: true
+      ENCRYPTION_KEY_DEV:
+        required: true
+
```

Comment:
```suggestion
```

Reply:
Solved in b3e1c006ace2fbbe6c0c90ba6c8a002d8b9cca21